### ISSUE TITLE
Tidy data for subject tables

### ISSUE THREAD

**Issue by arilamstein** [walkerke/tidycensus#108]
It seems to me that `tidycensus` might not be returning "tidy" data for subject tables. I might be wrong / misunderstanding what tidy data means or the goals of the package (i.e. maybe the package doesn't aspire to return data in a tidy format), but I thought to mention it here.

Consider this code and its output:
```
df = get_acs(geography="state", table="S1902")
df
> df
# A tibble: 2,808 x 5
   GEOID NAME    variable       estimate    moe
   <chr> <chr>   <chr>             <dbl>  <dbl>
 1 01    Alabama S1902_C01_001 1851061   5444  
 2 01    Alabama S1902_C01_002      72.5    0.2
 3 01    Alabama S1902_C01_003      16.3    0.2
 4 01    Alabama S1902_C01_004      35      0.2
 5 01    Alabama S1902_C01_005       6.7    0.1
 6 01    Alabama S1902_C01_006       1.8    0.1
 7 01    Alabama S1902_C01_007      21.1    0.2
 8 01    Alabama S1902_C01_008 1232880   5064  
 9 01    Alabama S1902_C01_009      18.9    0.2
10 01    Alabama S1902_C01_010      35.9    0.3
```

Here is the resource I am using to understand what "tidy" data means: https://blog.rstudio.com/2014/07/22/introducing-tidyr/.

In short, I expected `df` to have 50 rows (one for each state). I also expected each of the values of `unique(df$variable)` to in fact be columns of `'df`. This would allow one to type e.g. `summary(df$S1902_C01_001)` and find the average estimated value of that variable.

Things get complicated with ACS data because each "observation" in fact has two distinct values:  and estimate and a moe.

I am not an expert in tidyr and its functions, but my impression is that the package was designed to facilitate data transformations like this. I've spent a few hours experimenting with this, and thought to mention the idea here to get feedback.

Thanks again for the package.

**Comment by walkerke** [comment]
Hi @arilamstein - 

What you are describing is what I consider wide-form ACS data, which you can get from tidycensus functions with `output = "wide"`.  For example: 

```r
get_acs(geography = "state", table = "S1902", output = "wide")

# A tibble: 52 x 110
   GEOID NAME  S1902_C01_001E S1902_C01_001M S1902_C01_002E S1902_C01_002M
   <chr> <chr>          <dbl>          <dbl>          <dbl>          <dbl>
 1 01    Alab…        1851061           5444           72.5            0.2
 2 02    Alas…         250235           1399           86.3            0.4
 3 04    Ariz…        2448919           7179           74.7            0.2
 4 05    Arka…        1141480           3883           73.1            0.3
 5 06    Cali…       12807387          18852           80.4            0.1
 6 08    Colo…        2051616           4112           82.3            0.1
 7 09    Conn…        1354713           3509           79.2            0.2
 8 10    Dela…         348051           1568           76.9            0.4
 9 11    Dist…         276546           1294           80.5            0.4
10 12    Flor…        7393262          22885           72.1            0.1
# ... with 42 more rows, and 104 more variables: S1902_C01_003E <dbl>, ...
```

The long-form data returned by default by tidycensus is designed to work better with tidyverse tools (e.g. `group_by() %>% summarize()` would be the analogy to your example).  In the tidy model I use in tidycensus, the finest unit is not the geographic unit itself, but rather geography-variable combinations.  

I got a similar question a while back where I explained my design philosophy behind tidycensus, you can read it here: https://github.com/walkerke/tidycensus/issues/44#issuecomment-347299248
